web/src/resize.ts

```diff
@@ -221,25 +216,14 @@ function resize_navbar_alerts(): void {
     }
 }
 
-// On narrow screens, the `right` panel is absolutely positioned, so its
-// height doesn't change the height of `left` and vice versa. Here we
-// first let subheaders on both sides attain their natural height as
```

Comment:
Can you explain a bit more about why we need to do this for all screen widths now? It might be good to explain in a comment why this hacky code is needed instead of the html/CSS being able to handle this without us resizing.

Reply:
Preivously, both left and right subheader were in a single parent element, so we did not require it when both were visible as the height of parent was set accordingly. Now both subheaders are not in a single parent element.

Also added some explaination in comment.